### DISCUSSION TITLE
copilot icon is not showing in the vs code taskbar

### DISCUSSION THREAD

**Discussion by atif11802** [community/community#41965] [Copilot Conversations] [answered]
while working i saw an update of copilot . after reloading i saw there were no co pilot icon and suggestion is also not available.
in fact  already logged in through the vs code .what should i do now??

**Comment by detykoruqeho6** [comment]
The same question

**Comment by detykoruqeho6** [comment, marked as answer]
So I went back to the version from 3 days ago and the icon appeared again

**Reply by tamalCodes** [reply]
How do you rollback to a older version ?

**Reply by DinhHuynhTienPhu** [reply]
check this https://github.com/community/community/discussions/41959

**Reply by tamalCodes** [reply]
Done ! 
Thanks a tonn 🚀
Hope they fixes the issue soon

**Comment by jazzypants1989** [comment]


**Reply by DinhHuynhTienPhu** [reply]
use older version works for me, try the version from 3 days ago.

**Reply by barhouum7** [reply]
me too it just stopped showing on my vscode status bar, and I tried to reinstall older versions but still same issue
